assets/js/base/components/cart-checkout/return-to-cart-button/index.tsx

```diff
@@ -10,7 +10,7 @@ import { Icon, arrowBack } from '@woocommerce/icons';
  */
 import './style.scss';
 
-const ReturnToCartButton = ( { link } ) => {
+const ReturnToCartButton = ( { link }: { link?: string } ): JSX.Element => {
```

Comment:
Annoying I know, but I think it's preferred to create an interface for this. `interface ReturnToCartButtonProps` - I think it has a minor performance benefit, and also lets us refer to something by name, if we have to in future.